src/components/frame/Extensions/pages/AdvancedPage/AdvancedPage.tsx

```diff
@@ -4,81 +4,72 @@ import { FormattedMessage, useIntl } from 'react-intl';
 import NotificationBanner from '~common/Extensions/NotificationBanner';
 import { useColonyContext, useColonyContractVersion, useMobile } from '~hooks';
 import { canColonyBeUpgraded } from '~utils/checks';
-import Navigation from '~v5/common/Navigation';
-import TwoColumns from '~v5/frame/TwoColumns';
 import Button from '~v5/shared/Button';
 import ColonyVersionWidget from '~v5/shared/ColonyVersionWidget';
-import Spinner from '~v5/shared/Spinner';
 
 const displayName = 'frame.Extensions.pages.AdvancedPage';
 
 const AdvancedPage: FC = () => {
   const { formatMessage } = useIntl();
   const { colony } = useColonyContext();
   const { version } = colony || {};
-  const { colonyContractVersion, loadingColonyContractVersion } =
-    useColonyContractVersion();
+  const { colonyContractVersion } = useColonyContractVersion();
   const isMobile = useMobile();
 
   const canUpgrade = canColonyBeUpgraded(colony, colonyContractVersion);
 
   return (
-    <Spinner
-      loading={loadingColonyContractVersion}
-      loadingText={{ id: 'loading.advancedPage' }}
-    >
-      <TwoColumns aside={<Navigation pageName="extensions" />}>
-        <h3 className="heading-4 mb-6">
-          {formatMessage({ id: 'advancedPage.colony.title' })}
-        </h3>
-        <div className="mb-4">
-          <NotificationBanner
-            status={canUpgrade ? 'warning' : 'success'}
-            title={
-              <FormattedMessage
-                id={
-                  canUpgrade
-                    ? 'advancedPage.version.warning'
-                    : 'advancedPage.version.success'
-                }
-              />
-            }
-          />
-        </div>
-        <ColonyVersionWidget
-          currentVersion={colonyContractVersion}
-          lastVersion={version || 0}
-          status={canUpgrade ? 'error' : 'success'}
+    <div>
+      <h3 className="heading-4 mb-6">
+        {formatMessage({ id: 'advancedPage.colony.title' })}
+      </h3>
```

Comment:
Why not use the `FormattedMessage` and `Heading` components like we do for the rest of the CDapp?

Reply:
Just copied it. But will change.